src/Migrations/DatabaseLoader.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace UcanLab\LaravelDacapo\Migrations;
+
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Illuminate\Database\ConnectionInterface;
+
+class DatabaseLoader
+{
+    /**
+     * @var \Illuminate\Database\ConnectionInterface|\Illuminate\Database\Connection
+     */
+    private $connection;
+
+    /**
+     * @param ConnectionInterface $connection
+     */
+    public function __construct(ConnectionInterface $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * @throws
+     */
+    public function run(): void
+    {
+        $doctrine = $this->getDoctrineSchemaManager();
+
+        dump($doctrine->listTables());
```

Comment:
__[Phinder]__ Do not use dump. ([see on Sider](https://sider.review/gh/repos/151422991/pulls/95#issue-4831744))

<sub>posted by Sider</sub>